src/incomplete/rand.md

```diff
@@ -93,9 +93,10 @@ struct Color { // RGB Color struct
     b: f64,
 }
 
-// Implementing Rand for type Color
-impl Rand for Color {
-    fn rand<R: Rng>(rng: &mut R) -> Self {
+// Implement a Uniform random distribution for Color
+impl Distribution<Color> for Uniform {
+    #[inline]
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Point {
         Color {r: rng.next_f64(), b: rng.next_f64(), g: rng.next_f64()}
```

Comment:
`next_f64` doesn't exist any more

Reply:
Examples in `inclomplete` are not tested... Will fix.